src/unlimited.cpp

```diff
@@ -1382,7 +1382,11 @@ void BuildSeededBloomFilter(CBloomFilter& filterMemPool, std::vector<uint256>& v
     if (nTimePassed > nHoursToGrow * 3600)
         nFPRate = nMaxFalsePositive;
 
-    filterMemPool = CBloomFilter(nElements, nFPRate, insecure_rand(), BLOOM_UPDATE_ALL);
+    // Use a hardcoded seed value when not on mainnet so that we don't get false positives when running thinblock_tests.cpp
+    if (Params().NetworkIDString() == "main")
+        filterMemPool = CBloomFilter(nElements, nFPRate, insecure_rand(), BLOOM_UPDATE_ALL);
+    else
+        filterMemPool = CBloomFilter(nElements, nFPRate, 5, BLOOM_UPDATE_ALL);
```

Comment:
whenever we split what we test from what we run in the code, I get worried.  This seems to be very innocuous, but there are always possible issues like what if somebody makes a subsequent change to just one of these two lines by accident.

Is there any way we can push this into the test itself?  Maybe make it tolerant of false positives?